tools/integration_tests/run_integration_tests.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import logging
+import os
+from pathlib import Path
+
+from integration_test_utils import run_pysa_analysis, LOG
+
+
+def main() -> None:
+    """
+    Entry point function which checks if full_result.json is there, calls
+    functions from integration_test_utils to run pysa, parse
+    full_result.json, and compare the output.
+    """
+    if not os.path.isfile("./full_result.json"):
+        raise FileNotFoundError(
+            "./full_result.json file containing expected issues is not found"
+        )
+
+    logging.basicConfig(
+        level=logging.DEBUG,
+        format="%(asctime)s %(levelname)s %(message)s",
+    )
+
+    current_directory = Path(os.getcwd())
+
+    LOG.info("Running in `%s`", current_directory)
+    run_pysa_analysis(
+        current_directory,
+        passthrough_args="",
+        skip_model_verification=True,
+        filter_issues=False,
+        run_from_source=True,
+    )
+    LOG.info("Run produced expected results")
```

Comment:
should this go into `run_pysa_analysis`?

Reply:
Ya we can move it there!